mep/books/views.py

```diff
@@ -238,10 +240,18 @@ def get_context_data(self, **kwargs):
             description = 'By %s' % ','.join(
                 [a.name for a in self.object.authors])
         if self.object.year:
-            description += ', %s' % self.object.year
+            description += ', %s.\n ' % self.object.year
         if self.object.public_notes:
             description += self.object.public_notes
 
+        # text-only readable version of membership years for meta description
+        circ_years = strip_tags(as_ranges(self.object.event_years)
+                                .replace('</span>', ',')).rstrip(',')
```

Comment:
is this logic identical to the version that just got written to index membership year ranges for the map?

Reply:
yeah, we're using it in a couple of places now — didn't have a good thought for where to consolidate

Reply:
cool - idk either, but just wanted to check